source/imaer-shared/src/main/java/nl/overheid/aerius/shared/domain/v2/characteristics/adms/ADMSLimits.java

```diff
@@ -68,8 +68,10 @@
   public static final long SOURCE_MASS_FLUX_MAXIMUM = 100_000;
   public static final double SOURCE_MASS_FLUX_DEFAULT = 1;
 
-  public static final long SOURCE_L1_MINIMUM = 0;
+  public static final double SOURCE_L1_MINIMUM = 0.001;
   public static final long SOURCE_L1_MAXIMUM = 1_000;
+  public static final long SOURCE_L1_DEFAULT = 1;
```

Comment:
Do want to name these with `L1`? It's OK by me if the limits always work for that variable, but if it can differ based on the meaning of the variable...

Reply:
These limits are from the documentation where they are written as limits for `L1`. So I kept the reference to L1 for that reason.